utils.js

```diff
@@ -139,6 +139,33 @@ const SHARE_MESSAGES = [
   "OMG!",
 ];
 
+const RANDOM_MESSAGES = [
```

Comment:
Are these messages only showed on page refresh? The point of #40 is to show random but related messages when you refresh the page. I'm not sure how is that possible though.

Reply:
It seems like these messages are shown either way, not depending on the page refresh.

Reply:
Oh, sorry, I kinda understanded the issue some other way. I will correct it today or tomorrow and make a commit ASAP.

Reply:
Alright, I have done it. I think. You can check it now.

Reply:
Can we rename `RANDOM_MESSAGES` to `REFRESH_MESSAGES` to make it more explanatory?